### DISCUSSION TITLE
Permanent password after setting by PowerShell script not instantly active

### DISCUSSION THREAD

**Discussion by Runningman-work** [rustdesk/rustdesk#7204] [Q&A]
I have a PowerShell Script wich sets a permanent password after installation of RustDesk with option "--password". But it is only active after switching from "both passwords" to "use permanent password" (and back) in the client in the scrurity section, not instantly. What am I doing wrong ? Can I automate that somehow ?

regards,
Stefan

**Comment by Runningman-work** [comment]
problem resolved, installation script was not correct.

**Reply by rustdesk** [reply]
Where is no correct? We can correct it.

**Reply by Runningman-work** [reply]
Your script is ok, I used something like this in my script:

Start-Process "$env:ProgramFiles\RustDesk\RustDesk.exe" "--password $rustdesk_pw" -wait

and this line does not work, what works is this:

& 'C:\Program Files\RustDesk\rustdesk.exe' --password $rustdesk_pw
